README.md

```diff
@@ -133,32 +61,31 @@ Thank you to everyone contributing to Tauri!
 
 ### Documentation
 
-Documentation in a polyglot system is a tricky proposition. To this end, we prefer to use inline documentation of Rust code and at JSDoc in typescript / javascript code. We autocollect these and publish them using Docusaurus v2 and netlify. Here is the hosting repository for the documentation site: https://github.com/tauri-apps/tauri-docs
-
-### Testing & Linting
+Documentation in a polyglot system is a tricky proposition. To this end, we prefer to use inline documentation of Rust code and JSDoc in TypeScript / JavaScript code. We autocollect these and publish them using [starlight](https://starlight.astro.build/) and [netlify](https://netlify.com). Here is the hosting repository for the documentation site: https://github.com/tauri-apps/tauri-docs
```

Comment:
I don't believe we use JSDoc, just Typedoc. But could just remove the "JSDoc in" section.

I'd also remove the `We autocollect these and publish them using [starlight](https://starlight.astro.build/) and [netlify](https://netlify.com).` line and just keep the one to point to the docs repo where that respective info is since it's not really relevant to this repo.

Reply:
does this work? (stumbling over the wording...)

Reply:
Perfect, thanks!